packages/twenty-front/src/modules/settings/playground/components/PlaygroundPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { SettingsPath } from '@/types/SettingsPath';
+import {
+  PAGE_BAR_MIN_HEIGHT,
+  PageHeader,
+} from '@/ui/layout/page/components/PageHeader';
+import {
+  Breadcrumb,
+  BreadcrumbProps,
+} from '@/ui/navigation/bread-crumb/components/Breadcrumb';
+import styled from '@emotion/styled';
+import { useLingui } from '@lingui/react/macro';
+import '@scalar/api-reference-react/style.css';
+import { Button, IconX, useIsMobile } from 'twenty-ui';
+import { useNavigateSettings } from '~/hooks/useNavigateSettings';
+
+type PlaygroundPageProps = {
+  children: JSX.Element | JSX.Element[];
+  links: BreadcrumbProps['links'];
+};
+
+const StyledPage = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100dvw;
+  height: 100dvh;
+  background: ${({ theme }) => theme.background.noisy};
+`;
+
+const StyledMainContainer = styled.div<{ pageBarHeight: number }>`
+  padding-bottom: ${({ theme }) => theme.spacing(3)};
+  border-top: 1px solid ${({ theme }) => theme.border.color.medium};
+  height: calc(100vh - ${({ pageBarHeight }) => `${pageBarHeight}px`});
+  width: 100vw;
+`;
+
+export const PlaygroundPage = ({ children, links }: PlaygroundPageProps) => {
+  const isMobile = useIsMobile();
+  const { t } = useLingui();
+  const navigateSettings = useNavigateSettings();
+
+  const closePlayground = () => {
+    navigateSettings(SettingsPath.APIs);
+  };
+
+  return (
+    <StyledPage>
+      <PageHeader title={<Breadcrumb links={links} />}>
+        <Button
+          Icon={IconX}
+          dataTestId="close-button"
+          size={isMobile ? 'medium' : 'small'}
+          variant="secondary"
+          accent="default"
+          onClick={closePlayground}
+          ariaLabel={t`Close playground`}
```

Comment:
Let's do something a little more generic than playground and more like "Exit Full Screen".
We might want to re-use this in other places, for exemple the code editor in Workflow node would benefit from having a full screen mode (could be an interesting followup bounty for you)